src/django/api/tests.py

```diff
@@ -12,351 +12,431 @@
-        facility_list = FacilityList(header='address,country,name')
-        item = FacilityListItem(raw_data='1234 main st,Unknownistan,Shirts!',
-                                facility_list=facility_list)
-        parse_facility_list_item(item)
-        self.assert_failed_parse_results(
-            item, 'Could not find a country code for "Unknownistan".')
-
-
-class UserTokenGenerationTest(TestCase):
-    def setUp(self):
-        self.email = "test@example.com"
-        self.password = "example123"
-        self.user = User.objects.create(email=self.email)
-        self.user.set_password(self.password)
-        self.user.save()
-
-    def test_user_does_not_have_a_token_created_on_login(self):
-        login_response = self.client.post('/user-login/',
-                                          {"email": self.email,
-                                           "password": self.password},
-                                          format="json")
-        self.assertEqual(login_response.status_code, 200)
-        user = auth.get_user(self.client)
-        self.assertTrue(user.is_authenticated)
-        token = Token.objects.filter(user=self.user)
-        self.assertEqual(token.count(), 0)
-
-    def test_generated_token_is_not_deleted_on_logout(self):
-        self.client.login(email=self.email, password=self.password)
-        Token.objects.create(user=self.user)
-        logout_response = self.client.post('/user-logout/')
-        self.assertEqual(logout_response.status_code, 204)
-        user = auth.get_user(self.client)
-        self.assertFalse(user.is_authenticated)
-        token = Token.objects.filter(user=self.user)
-        self.assertEqual(token.count(), 1)
-
-
-class GeocodingUtilsTest(TestCase):
-    def setUp(self):
-        settings.GOOGLE_GEOCODING_API_KEY = "world"
-
-    def test_geocoding_api_url_is_created_correctly(self):
-        self.assertEqual(
-            create_geocoding_api_url("hello", "US"),
-            (
-                "https://maps.googleapis.com/maps/api/geocode/json"
-                "?components=country:US&address=hello&key=world"
-            )
-        )
-
-    def test_geocoded_address_data_is_formatted_correctly(self):
-        formatted_data = format_geocoded_address_data(
-            parsed_city_hall_data['full_response'])
-        self.assertEqual(formatted_data, parsed_city_hall_data)
-
-
-class GeocodingTest(TestCase):
-    def test_correct_address_is_geocoded_properly(self):
-        geocoded_data = geocode_address('City Hall, Philly', 'US')
-        self.assertDictEqual(geocoded_data, parsed_city_hall_data)
-
-    def test_ungeocodable_address_returns_value_error(self):
-        with self.assertRaises(ValueError) as cm:
-            geocode_address('hello world', '$#')
-
-        self.assertEqual(cm.exception.args, ('No results were found',))
-
-
-class FacilityListItemGeocodingTest(TestCase):
+# class FacilityListCreateTest(APITestCase):
```

Comment:
Wanted to check to make sure it was intentional to comment out all these tests.

Reply:
🤦‍♂️

Reply:
I restored the tests and rebased.